### ISSUE TITLE
why pretraining on 384x384 needs larger learning rate?

### ISSUE THREAD

**Issue by junwuzhang19** [keyu-tian/SparK#36]
Hi, thanks for your work. In general, for smaller batch size, a smaller learning rate is needed. Why SparK needs a larger learning rate with smaller batch size while pretraining on 384x384?

**Comment by keyu-tian** [comment]
Thanks. First note `--base_lr` is the **base learning rate**: the actual lr would be `base_lr * bs / 256`, as in [/pretrain/utils/arg_util.py line131](/pretrain/utils/arg_util.py#L131). And this lr scaling rule is commonly used in prior work like MAE.

So basically when we change the batch size, we don't need to care about `--base_lr`. It will adapt itself.

But, yes, actually we do double the `base_lr` when in 384 config, which is refer to BEiT. The reason could be the pretraining task in 384 are more difficult so we need a larger base_lr.
